# Hungarian translation for glib-networking.
# Copyright (C) 2011, 2012, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024. Free Software Foundation, Inc.
# This file is distributed under the same license as the glib-networking package.
#
# Gabor Kelemen <kelemeng at gnome dot hu>, 2011, 2012.
# Balázs Úr <ur.balazs at fsf dot hu>, 2012, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024.
msgid ""
msgstr ""
"Project-Id-Version: glib-networking master\n"
"Report-Msgid-Bugs-To: https://gitlab.gnome.org/GNOME/glib-networking/issues\n"
"POT-Creation-Date: 2024-02-15 20:54+0000\n"
"PO-Revision-Date: 2024-02-19 20:48+0100\n"
"Last-Translator: Balázs Úr <ur.balazs at fsf dot hu>\n"
"Language-Team: Hungarian <openscope at fsf dot hu>\n"
"Language: hu\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Lokalize 23.08.4\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: proxy/libproxy/glibproxyresolver.c:165
msgid "Proxy resolver internal error."
msgstr "Proxyfeloldó belső hiba."

#: tls/base/gtlsconnection-base.c:616 tls/base/gtlsinputstream.c:78
#: tls/base/gtlsinputstream.c:141 tls/base/gtlsoutputstream.c:78
#: tls/base/gtlsoutputstream.c:143
msgid "Connection is closed"
msgstr "A kapcsolat lezárva"

#. Cannot perform a blocking operation during a handshake on the
#. * same thread that triggered the handshake. The only way this can
#. * occur is if the application is doing something weird in its
#. * accept-certificate callback. Allowing a blocking op would stall
#. * the handshake (forever, if there's no timeout). Even a close
#. * op would deadlock here.
#.
#: tls/base/gtlsconnection-base.c:690
msgid "Cannot perform blocking operation during TLS handshake"
msgstr "A blokkoló művelet nem végezhető el a TLS-kézfogás alatt"

#: tls/base/gtlsconnection-base.c:755 tls/base/gtlsconnection-base.c:1303
#: tls/openssl/gtlsconnection-openssl.c:358
msgid "Socket I/O timed out"
msgstr "A foglalat túllépte az I/O időkorlátot"

#: tls/base/gtlsconnection-base.c:929
msgid "Server required TLS certificate"
msgstr "A kiszolgáló TLS-tanúsítványt kért"

#: tls/base/gtlsconnection-base.c:1519
#, c-format
msgid "Handshake is not finished, no channel binding information yet"
msgstr "A kézfogás nem fejeződött be, még nincsenek csatornakötési információk"

#: tls/base/gtlsconnection-base.c:1598
msgid "Peer does not support safe renegotiation"
msgstr "A partner nem támogatja a biztonságos újraegyeztetést"

#: tls/base/gtlsconnection-base.c:1746 tls/gnutls/gtlsconnection-gnutls.c:491
#: tls/openssl/gtlsconnection-openssl.c:209
#: tls/openssl/gtlsconnection-openssl.c:835
#, c-format
msgid "Unacceptable TLS certificate"
msgstr "Elfogadhatatlan TLS-tanúsítvány"

#: tls/base/gtlsconnection-base.c:2218
#, c-format
msgid "Receive flags are not supported"
msgstr "A fogadási jelzők nem támogatottak"

#: tls/base/gtlsconnection-base.c:2365
#, c-format
msgid "Send flags are not supported"
msgstr "A küldési jelzők nem támogatottak"

#: tls/gnutls/gtlscertificate-gnutls.c:297
#: tls/openssl/gtlscertificate-openssl.c:309
#, c-format
msgid "Failed to import PKCS #12: %s"
msgstr "Nem sikerült a PKCS #12 importálása: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:520
#: tls/openssl/gtlscertificate-openssl.c:519
#, c-format
msgid "Could not parse DER certificate: %s"
msgstr "A DER tanúsítvány nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:541
#: tls/openssl/gtlscertificate-openssl.c:540
#, c-format
msgid "Could not parse PEM certificate: %s"
msgstr "A PEM tanúsítvány nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:564
#: tls/openssl/gtlscertificate-openssl.c:561
#, c-format
msgid "Could not parse DER private key: %s"
msgstr "A DER személyes kulcs nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:587
#: tls/openssl/gtlscertificate-openssl.c:582
#, c-format
msgid "Could not parse PEM private key: %s"
msgstr "A PEM személyes kulcs nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:613
#, c-format
msgid "Could not import PKCS #11 certificate URI: %s"
msgstr "A PKCS #11 tanúsítvány URI nem importálható: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:657
#: tls/openssl/gtlscertificate-openssl.c:620
msgid "No certificate data provided"
msgstr "Nincsenek megadva tanúsítványadatok"

#: tls/gnutls/gtlscertificate-gnutls.c:981
#: tls/gnutls/gtlsconnection-gnutls.c:1058
#, c-format
msgid "Cannot verify peer identity of unexpected type %s"
msgstr "Nem lehet ellenőrizni a váratlan %s típus partner-személyazonosságát"

#: tls/gnutls/gtlsconnection-gnutls.c:204
msgid "Could not create TLS connection:"
msgstr "Nem sikerült létrehozni a TLS-kapcsolatot:"

#: tls/gnutls/gtlsconnection-gnutls.c:214
#: tls/gnutls/gtlsconnection-gnutls.c:234
#: tls/openssl/gtlsclientconnection-openssl.c:560
#: tls/openssl/gtlsserverconnection-openssl.c:455
#, c-format
msgid "Could not create TLS connection: %s"
msgstr "Nem sikerült létrehozni a TLS-kapcsolatot: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:435
#: tls/gnutls/gtlsconnection-gnutls.c:446
#: tls/gnutls/gtlsconnection-gnutls.c:460
#: tls/openssl/gtlsconnection-openssl.c:176
#, c-format
msgid "Peer failed to perform TLS handshake: %s"
msgstr "A partner nem tudta végrehajtani a TLS-kézfogást: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:468
msgid "TLS connection closed unexpectedly"
msgstr "A TLS-kapcsolat váratlanul befejeződött"

#: tls/gnutls/gtlsconnection-gnutls.c:483
#: tls/openssl/gtlsconnection-openssl.c:201
msgid "TLS connection peer did not send a certificate"
msgstr "A TLS kapcsolat partner nem küldött tanúsítványt"

#: tls/gnutls/gtlsconnection-gnutls.c:499
#, c-format
msgid "Peer sent fatal TLS alert: %s"
msgstr "A partner végzetes TLS riasztást küldött: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:509
msgid "Protocol version downgrade attack detected"
msgstr "Protokoll verzió visszaminősítéses támadás észlelve"

#: tls/gnutls/gtlsconnection-gnutls.c:518
#, c-format
msgid "Message is too large for DTLS connection; maximum is %u byte"
msgid_plural "Message is too large for DTLS connection; maximum is %u bytes"
msgstr[0] "Az üzenet túl nagy a DTLS kapcsolathoz; legfeljebb %u bájt lehet"
msgstr[1] "Az üzenet túl nagy a DTLS kapcsolathoz; legfeljebb %u bájt lehet"

#: tls/gnutls/gtlsconnection-gnutls.c:527
msgid "The operation timed out"
msgstr "A művelet túllépte az időkorlátot"

#: tls/gnutls/gtlsconnection-gnutls.c:879
#, c-format
msgid "Error performing TLS handshake: %s"
msgstr "Hiba a TLS-kézfogás végrehajtásakor: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:982
#: tls/openssl/gtlsconnection-openssl.c:614
msgid "Error performing TLS handshake"
msgstr "Hiba a TLS-kézfogás végrehajtásakor"

#: tls/gnutls/gtlsconnection-gnutls.c:1157
#, c-format
msgid ""
"Empty channel binding data indicates a bug in the TLS library implementation"
msgstr ""
"Az üres csatornakötési adatok programhibára utalnak a TLS programkönyvtár "
"megvalósításában"

#: tls/gnutls/gtlsconnection-gnutls.c:1175
#, c-format
msgid "Channel binding type is not implemented in the TLS library"
msgstr "A csatornakötési típus nincs megvalósítva a TLS programkönyvtárban"

#: tls/gnutls/gtlsconnection-gnutls.c:1179
#, c-format
msgid "Channel binding data is not yet available"
msgstr "A csatornakötési adatok még nem érhetők el"

#: tls/gnutls/gtlsconnection-gnutls.c:1231
#: tls/openssl/gtlsconnection-openssl.c:810
#, c-format
msgid "Requested channel binding type is not implemented"
msgstr "A kért csatornakötési típus nincs megvalósítva"

#: tls/gnutls/gtlsconnection-gnutls.c:1252
#: tls/gnutls/gtlsconnection-gnutls.c:1312
#: tls/openssl/gtlsconnection-openssl.c:828
#: tls/openssl/gtlsconnection-openssl.c:924
msgid "Error reading data from TLS socket"
msgstr "Hiba az adatok olvasásakor a TLS-foglalatból"

#. flags
#: tls/gnutls/gtlsconnection-gnutls.c:1334
#: tls/gnutls/gtlsconnection-gnutls.c:1397
#: tls/openssl/gtlsconnection-openssl.c:1003
msgid "Error writing data to TLS socket"
msgstr "Hiba az adatok TLS-foglalatba írásakor"

#: tls/gnutls/gtlsconnection-gnutls.c:1367
#, c-format
msgid "Message of size %lu byte is too large for DTLS connection"
msgid_plural "Message of size %lu bytes is too large for DTLS connection"
msgstr[0] "A(z) %lu bájt méretű üzenet túl nagy a DTLS kapcsolathoz"
msgstr[1] "A(z) %lu bájt méretű üzenet túl nagy a DTLS kapcsolathoz"

#: tls/gnutls/gtlsconnection-gnutls.c:1369
#, c-format
msgid "(maximum is %u byte)"
msgid_plural "(maximum is %u bytes)"
msgstr[0] "(legfeljebb %u bájt)"
msgstr[1] "(legfeljebb %u bájt)"

#: tls/gnutls/gtlsconnection-gnutls.c:1416
#, c-format
msgid "Error performing TLS close: %s"
msgstr "Hiba a TLS-lezárás végrehajtásakor: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:556
msgid ""
"Failed to load system trust store: GnuTLS was not configured with a system "
"trust"
msgstr ""
"Nem sikerült betölteni a rendszer megbízhatósági tárát: a GnuTLS nem "
"rendszer megbízhatósággal lett beállítva"

#: tls/gnutls/gtlsdatabase-gnutls.c:561 tls/openssl/gtlsdatabase-openssl.c:381
#, c-format
msgid "Failed to load system trust store: %s"
msgstr "Nem sikerült betölteni a rendszer megbízhatósági tárát: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:567
#, c-format
msgid ""
"System trust contains zero trusted certificates; please investigate your "
"GnuTLS configuration"
msgstr ""
"A rendszer megbízhatósága nulla megbízható tanúsítványt tartalmaz, vizsgálja"
" meg a GnuTLS beállításait"

#: tls/gnutls/gtlsdatabase-gnutls.c:583
#, c-format
#| msgid "Failed to populate trust list from %s: %s"
msgid "Failed to initialize trust list: %s"
msgstr "Nem sikerült előkészíteni a megbízhatósági listát: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:608
#, c-format
#| msgid "Failed to load system trust store: %s"
msgid "Failed to allocate credentials: %s"
msgstr "Nem sikerült kiosztani a hitelesítési adatokat: %s"

#: tls/gnutls/gtlsfiledatabase-gnutls.c:153
#: tls/openssl/gtlsfiledatabase-openssl.c:456
#, c-format
msgid "Failed to populate trust list from %s: %s"
msgstr "Nem sikerült feltölteni a megbízhatósági listát innen: %s: %s"

#: tls/gnutls/gtlsserverconnection-gnutls.c:124
#: tls/openssl/gtlsserverconnection-openssl.c:172
msgid "Certificate has no private key"
msgstr "A tanúsítványnak nincs személyes kulcsa"

#: tls/openssl/gtlsclientconnection-openssl.c:388
#: tls/openssl/gtlsserverconnection-openssl.c:299
#, c-format
msgid "Could not set TLS cipher list: %s"
msgstr "Nem sikerült beállítani a TLS titkosítási listát: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:416
#: tls/openssl/gtlsserverconnection-openssl.c:327
#, c-format
msgid "Could not set MAX protocol to %d: %s"
msgstr "Nem sikerült beállítani a MAX protokollt %d értékűre: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:505
#: tls/openssl/gtlsserverconnection-openssl.c:390
#, c-format
msgid "Could not create TLS context: %s"
msgstr "Nem sikerült létrehozni TLS-környezetet: %s"

#: tls/openssl/gtlsconnection-openssl.c:217
#, c-format
msgid "Unacceptable TLS certificate authority"
msgstr "Elfogadhatatlan TLS hitelesítésszolgáltató"

#: tls/openssl/gtlsconnection-openssl.c:225
msgid "Digest too big for RSA key"
msgstr "A kivonat túl nagy az RSA kulcshoz"

#: tls/openssl/gtlsconnection-openssl.c:234
msgid "Secure renegotiation is disabled"
msgstr "A biztonságos újraegyeztetés le van tiltva"

#: tls/openssl/gtlsconnection-openssl.c:253
#, c-format
msgid "%s: The connection is broken"
msgstr "%s: A kapcsolat megszakadt"

#: tls/openssl/gtlsconnection-openssl.c:660
#, c-format
msgid "The request is invalid."
msgstr "A kérés érvénytelen."

#: tls/openssl/gtlsconnection-openssl.c:683
#, c-format
msgid "Channel binding data tls-unique is not available"
msgstr "A tls-unique csatornakötési adatok nem érhetők el"

#: tls/openssl/gtlsconnection-openssl.c:706
#, c-format
msgid "X.509 Certificate is not available on the connection"
msgstr "Az X.509 tanúsítvány nem érhető el a kapcsolaton"

#: tls/openssl/gtlsconnection-openssl.c:714
#, c-format
msgid "Unable to obtain certificate signature algorithm"
msgstr "A tanúsítvány aláírási algoritmusa nem kérhető le"

#: tls/openssl/gtlsconnection-openssl.c:734
#, c-format
msgid ""
"Current X.509 certificate uses unknown or unsupported signature algorithm"
msgstr ""
"A jelenlegi X.509 tanúsítvány ismeretlen vagy nem támogatott aláírási "
"algoritmust használ"

#: tls/openssl/gtlsconnection-openssl.c:752
#, c-format
msgid "Failed to generate X.509 certificate digest"
msgstr "Nem sikerült előállítani az X.509 tanúsítványkivonatot"

#: tls/openssl/gtlsconnection-openssl.c:783
#, c-format
msgid "TLS Connection does not support TLS-Exporter feature"
msgstr "A TLS kapcsolat nem támogatja a TLS exportáló funkciót"

#: tls/openssl/gtlsconnection-openssl.c:786
#, c-format
msgid "Unexpected error while exporting keying data"
msgstr "Váratlan hiba a kulcsadatok exportálásakor"

#: tls/openssl/gtlsconnection-openssl.c:1064
msgid "Error performing TLS close"
msgstr "Hiba a TLS lezárás végrehajtásakor"

#: tls/openssl/gtlsdatabase-openssl.c:174
msgid "Could not get trust settings for certificate"
msgstr "Nem sikerült lekérni a tanúsítvány megbízhatósági beállításait"

#: tls/openssl/gtlsdatabase-openssl.c:261
msgid "Could not retrieve certificates"
msgstr "Nem sikerült lekérni a tanúsítványokat"

#: tls/openssl/gtlsdatabase-openssl.c:358
msgid "Could not get root certificate store"
msgstr "Nem sikerült lekérni a gyökér tanúsítványtárolót"

#: tls/openssl/gtlsdatabase-openssl.c:365
msgid "Could not get CA certificate store"
msgstr "Nem sikerült lekérni a CA tanúsítványtárolót"

#: tls/openssl/gtlsdatabase-openssl.c:430
msgid "Could not create CA store"
msgstr "Nem sikerült létrehozni a CA-tárat"

#: tls/openssl/gtlsserverconnection-openssl.c:184
#, c-format
msgid "There is a problem with the certificate: %s"
msgstr "Probléma van a tanúsítvánnyal: %s"

#: tls/openssl/gtlsserverconnection-openssl.c:193
#, c-format
msgid "There is a problem with the certificate private key: %s"
msgstr "Probléma van a tanúsítvány személyes kulcsával: %s"
